nixos/modules/programs/browserpass.nix

```diff
@@ -6,21 +6,17 @@ with lib;
 
   ###### interface
   options = {
-    programs.browserpass = {
-      enable = mkOption {
-        default = false;
-        type = types.bool;
-        description = ''
-          Whether to install the NativeMessaging configuration for installed browsers.
-        '';
-      };
-    };
+    programs.browserpass.enable = mkEnableOption "the NativeMessaging configuration for installed browsers.";
```

Comment:
Yes, I could not get it to work with firefox. b4caf7a11e8943e0263df41ba8dd9f21ff18f58e looks like it tried to add firefox support but from what I can see it does not affect the NixOS module.